spec/models/user_spec.rb

```diff
@@ -367,4 +366,21 @@
       it { expect(subject.admin?).to be_truthy }
     end
   end
+
+  describe '#full_name' do
+    subject { create(:user, first_name: 'first', last_name: 'last') }
+
+    it 'should compose #first_name' do
+      expect(subject.first_name).to eq('first')
+    end
+    it 'should compose #last_name' do
+      expect(subject.last_name).to eq('last')
+    end
```

Comment:
This method and the one above it are really testing the factory again.  Perhaps these should be in the section starting at line 12 as well.

Reply:
This was probably overkill; I will remove it for now.